src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,39 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+
+export const Tabs = ({ tabs, selectedTab, onTabSelected }) => {
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => {
+            const isSelected = tab.id === selectedTab.id;
```

Comment:
```suggestion
            const isSelectedTab = tab.id === selectedTab.id;
```

will be more informative